ReleaseChecklist.md

```diff
@@ -2,28 +2,32 @@
 
 ### Requirements
  - [ ] Lauchpad (Ubuntu One) account
- - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
+ - [ ] gnupg key (has to be version 1, gpg2 won't work) for ``your-name@ethereum.org`` created and uploaded
```

Comment:
Not entire clear what to do with that key... i.e. we could merge this with the previous point (Are gpg keys required for anything *but* launchpad? Nothing comes to my mind.)

Reply:
Good point. I think that's the only thing. Unless dockerhub also requires it (I don't have access to that) but I doubt it.

While at it I also added other requirements: all repos, npm, dockerhub, twitter, reddit, etc.

Reply:
I mean, if enough ppl have keys, we could start signing our commits?

Reply:
(just thinking about other cases where they are useful)

Reply:
Probably. I wonder how that works with rebases though. Doesn't this invalidate signatures?

Reply:
I think the sentiment in the past was that signed commits aren't really all that useful in general :-).